docs/developer-docs/gas-cost.md

```diff
@@ -153,7 +153,7 @@ The table below details the cost of compute, storage transmissions and canister
 | Canister creation | For creating canisters on a subnet. | Created canister | 100B | $0.133661 | 100B / 13 * 34 | $0.34957492307 |
 | Compute percent allocated per second | For each percent of the reserved compute allocation (a scarce resource). | Canister with allocation | 10M | $0.0000133661 | 10M / 13 * 34 | $0.00000511056 |
 | Update message execution | For every update message executed. | Target canister | 590K | $0.0000007885999 | 590K / 13 * 34 | $0.000000301523491 |
-| 1B executed instructions | For every 1B instructions executed when executing update type messages. | Canister executing instructions | 400M | $0.0053 | 400M / 13 * 34 | $0.00001572482 |
+| 1B executed instructions | For every 1B instructions executed when executing update type messages. | Canister executing instructions | 400M | $0.00053 | 400M / 13 * 34 | $0.0002044224 |
```

Comment:
Seems still wrong. On 34-node it should be more expensive: $0.00053 / 13 * 34 = $0.00138.

Actually almost all figures for this 34-node dollar amount seems wrong in this table.... is there an excel sheet for this somewhere? Maybe its formula should be adjusted